front/cypress/e2e/idea_edit_page.cy.ts

```diff
@@ -25,22 +25,24 @@ describe('Idea edit page', () => {
       })
       .then((project) => {
         projectId = project.body.data.id;
-        return cy.apiCreateIdea(
-          projectId,
-          ideaTitle,
-          ideaContent,
-          undefined,
-          undefined,
-          jwt
-        );
-      })
-      .then((idea) => {
-        ideaId = idea.body.data.id;
-        ideaSlug = idea.body.data.attributes.slug;
-        cy.wait(2000);
       });
   });
 
+  beforeEach(() => {
```

Comment:
I moved this piece of code to beforeEach such that the subsequent failures after the first fail for correct reasons and not an old ideaTitle.